nautobot/core/jobs/__init__.py

```diff
@@ -347,5 +366,161 @@ def run(self, *, content_type, csv_data=None, csv_file=None, roll_back_if_error=
             raise RunJobTaskFailed("CSV import not fully successful, see logs")
 
 
-jobs = [ExportObjectList, GitRepositorySync, GitRepositoryDryRun, ImportObjects, LogsCleanup, RefreshDynamicGroupCaches]
+class BulkEditObjects(Job):
```

Comment:
Let's add a new `nautobot.core.jobs.bulk_actions` submodule for this Job and the upcoming bulk-delete job, just to keep things a bit more organized.